larcorealg/Geometry/WireGeo.cxx

```diff
@@ -49,7 +51,7 @@ namespace geo{
     lp.SetZ(fHalfL);
     auto end = toWorldCoords(lp);
 
-    fThetaZ = std::acos((end.Z() - fCenter.Z())/fHalfL);
+    fThetaZ = std::acos(std::clamp((end.Z() - fCenter.Z())/fHalfL, -1.0, +1.0));
```

Comment:
I guess that one unfortunate byproduct of this is that the clamp would mask genuine problems (e.g. a bug causing the argument to genuinely evaluate to something much larger than 1). But, I guess it's not really the job of this function to catch such bugs...